website/docs/r/is_security_group_rule.html.markdown

```diff
@@ -105,7 +105,7 @@ Review the argument references that you can specify for your resource.
 - `icmp` - (Optional, List) A nested block describes the `icmp` protocol of this security group rule.
 
   Nested scheme for `icmp`:
-  - `type`- (Required, Integer) The ICMP traffic type to allow. Valid values from 0 to 254.
```

Comment:
type is also optional can we create icmp with out type ? or its taking default 0

Reply:
yes, please see the attachment:
```
type | integerexample: 8 | maximum: 254minimum: 0 | The ICMP traffic type to allow. If absent, all types are allowed.
```

<img width="675" alt="Screenshot 2022-12-01 at 11 01 09 AM" src="https://user-images.githubusercontent.com/91112322/204973854-abbf0361-bd1b-481f-b944-0a299005a1cf.png">

Reply:
@BShrivastav That is from our swagger but what is defined in Terraform resource schema. This documentation should be inline with Schema of that resource